test/test_examples_2d.jl

```diff
@@ -324,11 +324,16 @@ end
   end
   @testset "parameters_rotor.toml one step" begin
```

Comment:
```suggestion
  @testset "parameters_rotor.toml" begin
```
It's more than one step, isn't it?

Reply:
You are right, I forgot to change that

Reply:
Thanks!